src/renderer/base/RenderSettings.cpp

```diff
@@ -11,10 +11,6 @@
 using namespace Microsoft::Console::Render;
 using Microsoft::Console::Utils::InitializeColorTable;
 
-static constexpr size_t AdjustedFgIndex{ 16 };
-static constexpr size_t AdjustedBgIndex{ 17 };
-static constexpr size_t AdjustedBrightFgIndex{ 18 };

```

Comment:
Suppress white space changes if you haven't done so already. It'll effectively remove almost all changes in this file.